### PR TITLE
OSDOCS#12882:Add cluster after pausing the hosted cluster reconciliation

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-12882

Link to docs preview: https://86051--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp_high_availability/hcp-disaster-recovery-oadp.html#hcp-dr-oadp-backup-cp-workload_hcp-disaster-recovery-oadp

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-dr-oadp-backup-cp-workload.adoc [hunk 1]
```diff
@@ -21,6 +21,24 @@ $ oc --kubeconfig <management_cluster_kubeconfig_file> \
...
+$ oc get hostedcluster -n local-cluster <hosted_cluster_name> -o=jsonpath="{.spec.infraID}"
+----
++
+Note the infrastructure ID to use in the next step.
+
+. Pause the reconciliation of the `HostedCluster` resource by running the following command:
++
+[source,terminal]
+----
+$ oc patch cluster \
```
Comment (LiangquanLi930) [inline modules/hcp-dr-oadp-backup-cp-workload.adoc:37]: should be change to
```
$ oc patch cluster.cluster.x-k8s.io \
  -n local-cluster-<hosted_cluster_name> <hosted_cluster_infra_id> \
  --type json -p '[{"op": "add", "path": "/spec/paused", "value": true}]'
```

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**